_scripts/fetch-upstream-resources.sh

```diff
@@ -59,33 +59,25 @@ DISTRIBUTION_BRANCH="release/2.7"
 
 # Directories to get via SVN. We use this because you can't use git to clone just a portion of a repository
 svn co https://github.com/docker/docker-ce/"$ENGINE_SVN_BRANCH"/components/cli/docs/extend ./engine/extend || (echo "Failed engine/extend download" && exit 1)
-svn co https://github.com/docker/docker-ce/"$ENGINE_SVN_BRANCH"/components/engine/docs/api ./engine/api || (echo "Failed engine/api download" && exit 1) # This will only get you the old API MD files 1.18 through 1.24
+svn co https://github.com/docker/docker/branches/master/docs/api ./engine/api || (echo "Failed engine/api download" && exit 1) # This will only get you the old API MD files 1.18 through 1.24
```

Comment:
This can be changed back to the release branch, once https://github.com/moby/moby/pull/40575 is merged